packages/core/admin/admin/src/pages/MarketplacePage/components/PluginCard/index.js

```diff
@@ -73,18 +93,42 @@ const PluginCard = ({ plugin }) => {
             { pluginName: attributes.name }
           )}
           variant="tertiary"
+          onClick={() => trackUsage('didLearnMore')}
```

Comment:
Should be name the event 
```suggestion
          onClick={() => trackUsage('didPluginLearnMore')}
```

so that it's clear it's about plugins?

Reply:
Good point, I'm asking the team.